services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/tormaystarkastelu/TormaystarkasteluLaskentaService.kt

```diff
@@ -17,6 +17,17 @@ class TormaystarkasteluLaskentaService(
     @Autowired private val tormaysService: TormaystarkasteluTormaysService
 ) {
 
+    companion object {
```

Comment:
According to https://kotlinlang.org/docs/coding-conventions.html#class-layout companion objects should be near the bottom of the class.

Reply:
Moved to bottom